packages/devtools/lib/src/timeline/flame_chart_canvas.dart

```diff
@@ -248,6 +251,14 @@ class FlameChartCanvas extends FlameChart {
       shouldCollapseNativeSamples = shouldCollapse;
       _viewportCanvas.rebuild(force: true);
     });
+
+    final measurementCanvas = CanvasElement().context2D
```

Comment:
move this to an _initAsciiMeasurements method

Reply:
done.